Dockerfile

```diff
@@ -144,6 +144,7 @@ RUN \
   sed -i \
     's|#ssl_trusted_certificate /config/keys/cert.crt;|ssl_trusted_certificate /config/keys/cert.crt;|' \
     /defaults/nginx/ssl.conf.sample && \
+  rm -f /etc/nginx/conf.d/stream.conf && \
```

Comment:
Could we have a comment above this just so it's not implied to be part of the `enable OCSP stapling from base` lines above it?

Reply:
Done